### PR TITLE
large headers

### PR DESCRIPTION
**Pull request by bloodearnest**
- Improve local testing experience
- Bump max proxy header size

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,3 +50,11 @@ Full integraton tests can only be run against the current production
 deployment, as it requires TLS and DNS:
 
     ./full-tests.sh
+
+## Debug build
+
+Ir you `export IMAGE_NAME=proxy-debug`, just run/test will now use build of the
```
Comment (madwort) [inline README.md:56]:
```suggestion
If you `export IMAGE_NAME=proxy-debug`, just run/test will now use build of the
```

File: Dockerfile [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
 ENV PORT=8080
 # Default orgs, must be pipe separated as are fed into a regex match
 # Can be overridden via run time env var
 ENV ORGS=opensafely|opensafely-core|opensafely-actions
 COPY *.conf.template /etc/nginx/templates/
-# uncomment to build a debug version locally
-#CMD ["nginx-debug", "-g", "daemon off; error_log /var/log/nginx/error.log debug;"]
+
+
+FROM proxy AS proxy-debug
```
Comment (madwort) [inline Dockerfile:9]:
```suggestion
FROM opensafely-proxy AS opensafely-proxy-debug
```

File: README.md [hunk 3]
```diff
@@ -50,3 +50,11 @@ Full integraton tests can only be run against the current production
 deployment, as it requires TLS and DNS:
 
     ./full-tests.sh
+
+## Debug build
+
+If you `export IMAGE_NAME=proxy-debug`, just run/test will now use build of the
```
Comment (madwort) [inline README.md:56]:
```suggestion
If you `export IMAGE_NAME=opensafely-proxy-debug`, just run/test will now use build of the
```

Comment (bloodearnest) [reply README.md:56]: I kept the local name as just proxy-debug, as wanted to avoid any sense of it being publishable.

File: Dockerfile (hunk 2 above)
Comment (bloodearnest) [reply Dockerfile:9]: Have fixed the bug, but kept the local-only name as `proxy-debug`

**Review by madwort — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (madwort) [reply Dockerfile:9]: 🤝

File: README.md (hunk 3 above)
Comment (madwort) [reply README.md:56]: fair enough

**Pull request merged**